frontend/texts/postulate-1.yc

```diff
@@ -8,7 +8,7 @@ Again, since the [point B] is the center of the [circle CAE name=ACE], therefore
 But [line AC] was proved equal to [line AB], therefore each of the straight lines [line AC] and [line BC] equals [line AB].
 And things which equal the same thing also equal one another, therefore [line AC] also equals [line BC].
 Therefore the three straight lines [line AC], [line AB], and [line BC] equal one another.
-Therefore the [polygon ABC] is equilateral, and it has been constructed on the given finite straight [line AB].
+Therefore the [polygon ABC color=#ffffffff] is equilateral, and it has been constructed on the given finite straight [line AB].
```

Comment:
Can we also support `#ffffff` with no alpha value? Then we can just assume alpha=1.